BUILDING.md

```diff
@@ -635,8 +636,16 @@ disk space.
 
 #### Building Node.js
 
-If the path to your build directory contains a space or a non-ASCII character,
-the build will likely fail.
+* Remember to first clone the Node.js repository with the Git command
+  and head to the directory that Git created; If you haven't already
+  ```powershell
+  git clone https://github.com/nodejs/node.git
+  cd node
```

Comment:
I dunno if this works in powershell, but with shell and mkdir, you can combine with the `$_` like `mkdir foo && cd $_` to immediately switch to the newly created thing.

Reply:
It does, but as the other PowerShell commands also use multiline convention